java/java.editor/test/unit/src/org/netbeans/modules/java/editor/imports/ClipboardHandlerTest.java

```diff
@@ -179,6 +180,19 @@ private void copyAndPaste(String from, final String to, String golden, String so
         assertEquals(golden, actual[0]);
     }
 
+    // NbClipboard is asynchronoous - this test has to wait for copy/paste to finish
+    private static void syncNbClipboard() {
+        NbClipboard nbclip = Lookup.getDefault().lookup(NbClipboard.class);
+        assertNotNull("active NbClipboard expected", nbclip);
+        try {
```

Comment:
Calling getContents() might be better here?

Reply:
blocking on the tasks is the most-direct way to sync on the clipboard. The method is also used from another test.

Reply:
So is getContents. I don't think we should change this test, and if we do I think it should use the API.